UPGRADE.md

```diff
@@ -352,59 +352,42 @@ All react-admin components now accept a `className` prop instead of the `elStyle
 + export default withStyles(styles)(UserList);
 ```
 
-In addition to `elStyle`, Field and Input components used to support a `style` prop to override the styles of the *container element* (the `<td>` in a datagrid). This prop is no longer supported in react-admin. Instead, the `Datagrid` component will check if its children have a `headerClassName` and `cellClassName` props. If they do, it will apply those classes to the table header and cells respectively.
-
-```jsx
-// before
-import { EmailField, List, Datagrid } from 'react-admin';
+Yes, it's way longer to write and it's a pain, but that's material-ui's choice of CSS-in-JS implementation - and there is nothing the react-admin team can do about it. If this is a consolation, we suffered to upgrade our own apps to JSS, too.
```

Comment:
Not sure we should add that

Reply:
You're right, we can mention it in a blog post, not in an upgrade guide.